src/polynomial.cpp

```diff
@@ -0,0 +1,310 @@
+#include "polynomial.h"
+#include "add.h"
+#include "mul.h"
+#include "pow.h"
+#include "constants.h"
+
+namespace SymEngine {
+
+    Polynomial::Polynomial(const RCP<const Symbol> &var, map_uint_mpz&& dict) :
```

Comment:
I am going to shift it back by 4 spaces to maintain uniformity across SymEngine. @isuruf